__tests__/app.test.js

```diff
@@ -56,4 +57,33 @@ describe("API", () => {
         });
     });
   });
+
+  //GET /API/ARTICLES
+  describe("GET/api/articles - Display Array of article objects", () => {
+    it("200: GET responds with an array of article objects, each of which to have appropriate properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          console.log(body);
+          expect(body.articles).toBeInstanceOf(Array);
+          expect(body.articles).toHaveLength(12);
+          expect(body.articles).toBeSortedBy("created_at", {
```

Comment:
Awesome use of the sorted by package here to check that the articles are in the correct order